cache-config/testing/ort-tests/tcdata/todb.go

```diff
@@ -148,6 +148,8 @@ func SetupRoleCapabilities(db *sql.DB) error {
 	sqlStmt := `
 INSERT INTO role_capability (role_id, cap_name) VALUES (4,'all-write') ON CONFLICT DO NOTHING;
 INSERT INTO role_capability (role_id, cap_name) VALUES (4,'all-read') ON CONFLICT DO NOTHING;
+INSERT INTO role_capability (role_id, cap_name) VALUES (3,'all-read') ON CONFLICT DO NOTHING;
+INSERT INTO role_capability (role_id, cap_name) VALUES (3,'all-write') ON CONFLICT DO NOTHING;
```

Comment:
These don't actually do anything unless you set them to explicitly in your Traffic Ops configuration file (and it isn't in these tests). In which case, these specific strings (`all-read` and `all-write`) still have no effect and no meaning.